### ISSUE TITLE
add test for 'ipfs files mkdir -p'

### ISSUE THREAD

**Issue by whyrusleeping** [dignifiedquire/randor#19]
We want to stress mkdir with and without the `-p` option. And also want to make sure that both get called on directories that already exist (to assert that nothing happens, and their contents don't change).

Also want to make sure we run `mkdir -p` on paths that already partially exist (i.e. `mkdir -p a/b/c/d`when a/b already exists)

**Comment by dignifiedquire** [comment]
whaaat mkdir has the p flag?? And there I was generating this behaviour manually...

**Comment by dignifiedquire** [comment]
Doing this manually without the `-p` flag right now: https://github.com/dignifiedquire/randor/blob/master/lib/operations/files-write.js#L41

Testing `mkdir` on dirs that already exists is a bit annoying because of the error thrown, but can be done.

**Comment by whyrusleeping** [comment]
Yeah, using mkdir -p on dirs that already exists won't throw an error, And thats the operation i'm more interested in
